--- a/youtube_dl_webui/db.py     2018-04-11 10:15:12.000000000 +0200
+++ b/youtube_dl_webui/db.py     2020-04-10 19:03:30.737928000 +0200
@@ -248,19 +248,29 @@
 
     def update_info(self, tid, info_dict):
         self.logger.debug('db update_info()')
-        db_data =   {
-                        'valid':            1,      # info_dict is updated
-                        'title':            info_dict['title'],
-                        'format':           info_dict['format'],
-                        'ext':              info_dict['ext'],
-                        'thumbnail':        info_dict['thumbnail'],
-                        'duration':         info_dict['duration'],
-                        'view_count':       info_dict['view_count'],
-                        'like_count':       info_dict['like_count'],
-                        'dislike_count':    info_dict['dislike_count'],
-                        'average_rating':   info_dict['average_rating'],
-                        'description':      info_dict['description'],
-                    }
+        
+        db_data =   {'valid':            1}
+        if 'title' in info_dict :
+            db_data['title'] = info_dict['title']
+        if 'format' in info_dict :
+            db_data['format'] = info_dict['format']
+        if 'ext' in info_dict :
+            db_data['ext'] = info_dict['ext']
+        if 'thumbnail' in info_dict :
+            db_data['thumbnail'] = info_dict['thumbnail']
+        if 'duration' in info_dict :
+            db_data['duration'] = info_dict['duration']
+        if 'view_count' in info_dict :
+            db_data['view_count'] = info_dict['view_count']
+        if 'like_count' in info_dict :
+            db_data['like_count'] = info_dict['like_count']
+        if 'dislike_count' in info_dict :
+            db_data['dislike_count'] = info_dict['dislike_count']
+        if 'average_rating' in info_dict :
+            db_data['disaverage_ratinglike_count'] = info_dict['average_rating']
+        if 'description' in info_dict :
+            db_data['description'] = info_dict['description']
+
         self.update(tid, {'task_info': db_data})
 
     def update_log(self, tid, log, exist_test=False):
